components/prism-solidity.js

```diff
@@ -0,0 +1,77 @@
+Prism.languages.solidity = Prism.languages.extend('clike', {
+    'keyword': [
+		{pattern: /\b(?:var|import|function|constant|view|pure|payable|storage|memory|if|else|for|while|do|break|continue|returns?|private|public|internal|external|inherited|this|suicide|selfdestruct|emit|new|is|throw|revert|assert|require|\_)\b/},
```

Comment:
You don't need to use an object here, since you only need the `pattern` key. You can provide the regexp directly.
Also, you might want to reorder the keywords alphabetically for easier additions/removal in the future.
Finally, you don't need to escape the `_`.